meteor/client/ui/Settings/components/triggeredActions/actionEditors/filterPreviews/AdLibFilter.tsx

```diff
@@ -287,15 +298,39 @@ export const AdLibFilter: React.FC<IProps> = function AdLibFilter({
 		'type',
 	]
 
+	const availableOptions = useTracker<Record<string, any> | string[], Record<string, any> | string[]>(
+		() => {
+			// tags are a special case because we need to search the database for available options
+			// we should have the data subscribed already
+			if (link.field === 'tag') {
+				return _.chain(
+					RundownBaselineAdLibActions.find()
```

Comment:
It might be more readable to do
```
[
  ...RundownBaselineAdllibActions.find().map(...),
  ...AdLibActions.find().map(...),
  ...RundownBaselineAdLibPieces.find().map(...),
  ...AdLibPieces.find().map(...),
]
```

Reply:
Thanks, that's a good suggestion.